### ISSUE TITLE
Toggle between Original and Groomed Surfaces does not work

### ISSUE THREAD

**Issue by patkins** [SCIInstitute/ShapeWorks#1730]
When opening an existing project, either on the Groom or Optimize panel, the toggle of what shapes are shown does not work as expected. When 'original' is shown, the 'groomed' orientation of the shapes appears. I noticed this in ShapeWorks 6.2.1, but have confirmed that the functionality remains in the current development build (ShapeWorks-dev-5772-ee01c2989-linux).

I have uploaded an example project here: /home/sci/atkins/SWIssues/pelvis which can be opened as is, then using the toggle between original and groomed (noting no difference), then running ICP alignment which alters the alignment of the three shapes significantly and then toggling between original and groom (and still noting no difference).

**Comment by akenmorris** [comment]
@patkins, This should be fixed in the latest development build.  The bug was actually related to groom parameters for single domain runs.  I'm not sure exactly how a project gets into this state but there are two groom columns, one default and one for the single domain.  The UI controls are using one and the execution of groom was using the other.  So your grooming wasn't doing what you expected.  Please try out the development release and confirm.

**Comment by patkins** [comment]
@akenmorris I tried with the latest linux dev build and still seem to have the same issue. I started a new project, imported three shapes, used the ICP alignment which obviously altered the alignment, and then toggled between Original and Groomed and see only the aligned shapes for both.

**Comment by akenmorris** [comment]
Right, separate issue.  So alignment is treated a little differently.  We have the "align" checkbox below the viewers.  Before there is any grooming alignment, this just does centering.  The reason to do this prior to grooming is so that the user can see all of their shapes at once.  After grooming, this `align` checkbox should use the grooming alignment, be it centering or ICP.  This will apply regardless of the view type (original/groomed).  This allows the user to apply planes and free form constraints on the original data using the alignments and also to more easily compare grooming results (e.g. smoothing and remeshing) against the original.

**Comment by patkins** [comment]
Got it, that makes sense. I was just used to to 'old' functionality. I'm closing this issue.
